packer/cache.go

```diff
@@ -5,16 +5,12 @@ import (
 	"path/filepath"
 )
 
-var DefaultCacheDir = "packer_cache"
-
 // CachePath returns an absolute path to a cache file or directory
 //
-// When the directory is not absolute, CachePath will try to get
-// current working directory to be able to return a full path.
-// CachePath tries to create the resulting path if it doesn't exist.
-//
-// CachePath can error in case it cannot find the cwd.
+// When the directory is not absolute, CachePath will try to make a
+// a cache depending on the operating system.
 //
```

Comment:
👋🏼 
Could you please update the examples down here in this list depending on the OS ? That would make it easier for me. 🙂

Reply:
Updated.

Reply:
Super awesome, thanks.